Jenkinsfile.conda

```diff
@@ -0,0 +1,55 @@
+pipeline {
+    agent any
+    environment {
+        idl_version = "${GIT_TAG_NAME}"
+        xml_version = "4.6.0"
+        dockerImageName = "lsstts/conda_package_builder:latest"
+        container_name = "idl_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
+    }
+
+    stages {
+        stage("Pull Docker Image") {
+            steps {
+                script {
+                    sh """
+                    docker pull ${dockerImageName}
+                    """
+                }
+            }
+        }
+        stage("Create IDL Conda package") {
+            steps {
+                script {
+                    sh """
+                    docker run --name ${container_name} -di --rm -v ${WORKSPACE}/conda:/home/saluser/conda ${dockerImageName}
```

Comment:
Inject `idl_version` and `xml_version` when running the container.  e.g. 

```
--env TS_XML_VERSION=v${xml_version} --env idl_version=${idl_version}
```

Reply:
Fixed in commit a27266d0ca07c4a24761d164fcef795645fe1aa4.